akka-http-core/src/main/scala/akka/http/scaladsl/model/HttpEntity.scala

```diff
@@ -623,7 +623,7 @@ object HttpEntity {
       private var bytesLeft = Long.MaxValue
 
       override def preStart(): Unit = {
-        attributes.getFirst[SizeLimit] match {
+        attributes.get[SizeLimit] match {
```

Comment:
Hmm, some test failures around this features - is `get` really a good replacement for `getFirst`?

Reply:
Maybe not? Just followed the deprecation instructions to see where that leads us. Removing that commit for now...